x/nft/msg.go

```diff
@@ -0,0 +1,56 @@
+package nft
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+)
+
+var _ weave.Msg = (*AddApprovalMsg)(nil)
+var _ weave.Msg = (*RemoveApprovalMsg)(nil)
+
+const (
+	pathAddApproval    = "nft/add-approval"
```

Comment:
pp: we should have a convention for naming. I used `username/address/add` for example. I am open to discuss this.

Reply:
that will do